src/tools/kvstore_tool.cc

```diff
@@ -49,6 +49,8 @@ StoreTool::StoreTool(const string& type,
   }
 }
 
+#ifdef WITH_BLUESTORE
```

Comment:
Wouldn't it be better to leave load_bluestore defined unconditionally but remove relevant #ifdef..endif from StoreTool ctor and put it inside load_bluestore() func?
int StoreTool::load_bluestore()
{
#ifdef WITH_BLUESTORE
...
#else
   cerr << "bluestore not compiled in" << std::endl;
   exit(1);
#endif
}

Reply:
Maybe, but you don't have to convince me, because my patch only fixes the build failure but does not refactor anything.
It was @liewegas who decided for this structure in commit 732ad1364e8693945641f2435500dfa475f0b58f